packages/fx-core/src/core/index.ts

```diff
@@ -1363,13 +1371,22 @@ export class FxCore implements Core {
           ? await v1plugin.getQuestions(Stage.create, context as SolutionContext)
           : ok(undefined);
       }
-      if (res.isErr()) return res;
+      if (res.isErr()) return err(new SystemError(res.error, CoreSource, "QuestionModelFail"));
       if (res.value) {
-        const solutionNode = res.value as QTreeNode;
-        solutionNode.condition = { equals: solutionPlugin.name };
-        if (solutionNode.data) solutionSelectNode.addChild(solutionNode);
+        const solutionNode = Array.isArray(res.value)
+          ? (res.value as QTreeNode[])
+          : [res.value as QTreeNode];
+        for (const node of solutionNode) {
```

Comment:
Currently the solution node might be an array and each solution can return several nodes